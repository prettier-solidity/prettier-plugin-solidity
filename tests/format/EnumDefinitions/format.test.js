runFormatTest(import.meta, ['solidity-parse']);
runFormatTest(import.meta, ['solidity-parse'], { bracketSpacing: true });
