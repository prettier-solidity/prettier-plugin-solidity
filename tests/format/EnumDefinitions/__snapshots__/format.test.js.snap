// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`EnumDefinitions.sol format 1`] = `
====================================options=====================================
parsers: ["slang"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
contract EnumDefinitions {
    enum ActionChoices { GoLeft, GoRight, GoStraight, SitStill }
    enum States { State1, State2, State3, State4, State5, State6, State7, State8, State9 }
    enum Rounding {
    Floor, // Toward negative infinity
    Ceil, // Toward positive infinity
    Trunc, // Toward zero
    Expand // Away from zero
    }

}

=====================================output=====================================
contract EnumDefinitions {
    enum ActionChoices {
        GoLeft,
        GoRight,
        GoStraight,
        SitStill
    }
    enum States {
        State1,
        State2,
        State3,
        State4,
        State5,
        State6,
        State7,
        State8,
        State9
    }
    enum Rounding {
        Floor, // Toward negative infinity
        Ceil, // Toward positive infinity
        Trunc, // Toward zero
        Expand // Away from zero
    }
}

================================================================================
`;
